### PR TITLE
Add rake task for publishing special routes.

### PR DESCRIPTION
**Pull request by elliotcm**
Includes things like /robots.txt

https://trello.com/c/blLdEZN5/292-make-apps-register-special-routes-on-deploy

### CODE REVIEW

File: lib/tasks/router.rake [hunk 1]
```diff
@@ -13,33 +13,6 @@ namespace :router do
...
-        raise
-      end
-    end
-
-    @router_api.commit_routes
-  end
-
   desc "Register application and routes with the router"
-  task :register => [ :register_backend, :register_routes ]
+  task :register => :register_backend
```
Comment (alext) [inline lib/tasks/router.rake]: `register_backend` shouldn't be needed any more now that this isn't registering any routes.  It shouldn't be talking directly to the router at all.

Comment (elliotcm) [inline lib/tasks/router.rake]: Does the backend get registered each time by the content store?

Comment (alext) [inline lib/tasks/router.rake]: Yep - https://github.com/alphagov/content-store/blob/master/app/models/registerable_route_set.rb#L76

Comment (elliotcm) [inline lib/tasks/router.rake]: Cool.  Updated.

File: app/models/special_route.rb [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+                    such as developers interested in joining GDS.",
+    },
+    {
+      base_path: "/fonts",
+      content_id: "f0115875-227e-4441-a061-3fdc6d79b7e6",
+      title: "Fonts prefix",
+      description: "This location previously held fonts for GOV.UK, but has since been removed.",
+      type: "prefix",
+      rendering_app: "gone",
+    },
```
Comment (alext) [inline app/models/special_route.rb]: I'm not sure it's worth including this in here.  It was never actually live on the www.gov.uk domain, and was originally included by mistake (the fonts are all served through the assets domain).

**Comment by alext** [comment]
This will require updates to the hooks in the deploy scripts to stop calling the `router:register` task, and instead to call the `publishing_api:publish_special_routes` task.

**Comment by elliotcm** [comment]
@alext The relevant PR is here, will need modifying now the router task is gone: https://github.gds/gds/alphagov-deployment/pull/1019

File: app/models/special_route.rb (hunk 2 above)
Comment (elliotcm) [inline app/models/special_route.rb]: Removed.

**Comment by elliotcm** [comment]
@alext Updated to use the gem and include an alias avoiding alphagov deployment changes.

File: test/test_helper.rb [hunk 3]
```diff
@@ -6,3 +6,4 @@
 require 'test/unit'
 require 'rails/test_help'
 require 'mocha/setup'
+require 'webmock/minitest'
```
Comment (danielroseman) [inline test/test_helper.rb]: Why is this here? It seems unrelated.

Comment (elliotcm) [inline test/test_helper.rb]: Leftover from the previous implementation.  Removed.
